apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateEntitiesFromDslMojoIT/expected/FissClaim.java

```diff
@@ -231,13 +228,13 @@ public class FissClaim {
       size = 100
   )
   @Builder.Default
-  private List<FissPayer> payers = new LinkedList<>();
+  private Set<FissPayer> payers = new HashSet<>();
 
   @ManyToOne(
       fetch = FetchType.EAGER
   )
   @JoinColumn(
-      name = "`mbiId`"
+      name = "`mbi_id`"
```

Comment:
do we need the back-ticks now that the db columns are snake_case?

Reply:
This was a test case verifying that the feature works.  Might be worthwhile to remove the feature in a later PR.